core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -760,6 +762,29 @@ oppia.directive('conversationSkin', [
                 var nodeId = UrlService.getUrlParams().node_id;
                 StoryViewerBackendApiService.recordStoryNodeCompletion(
                   storyId, nodeId);
+
+                StoryViewerBackendApiService.fetchStoryData(storyId).then(
+                  function(storyDataDict) {
+                    var storyNodes = storyDataDict.story_nodes;
```

Comment:
We should be converting this to a domain object? I would expect something like

    var storyNodes = storyDataDict.story_nodes.map(function(storyNodeDict)  {
      return StoryNodeObjectFactory.createFromBackendDict(storyNodeDict);
    });

Reply:
Done.